src/store/storage-controller.ts

```diff
@@ -28,15 +29,17 @@ export class StorageController {
     indexes: { [key: string]: string },
     dataStream?: Readable
   ): Promise<void> {
+    const recordId = Message.getRecordId(message);
```

Comment:
Looking at this, your idea of using `messageCid` as opposed to the original intent of "logical ID' probably ends up being a better idea after all..... 

The problem with directly referencing of `recordId` here is that the storage controller is intended be generic and thus able to handle storing of data regardless of message being a Record, or a Protocol Definition, or a PermissionGrant or HookDefinition whatnot. Which is why I named the ID `logicalId` and not `recordId`, but if we follow that pattern, we'd need to have a method somewhere called `getLogicalId()`.

But your idea of using `messageCid` would sidestep the above and still work.... do you mind switching to `messageCid` given the above reasoning?

Reply:
LOL sure yeah i can go back to using `messageCid` :)